src/utilities/useGeoJS.ts

```
@@ -0,0 +1,41 @@
+import {
+  ref, onMounted, Ref,
+} from '@vue/composition-api';
+
+import geo from 'geojs';
+
+export default function useGeoJS(element: Ref<HTMLElement | null>) {
+  const map: Ref<any> = ref(null);
+  const zoomLevel = ref(0);
+  const xCoord = ref(0);
+  const yCoord = ref(0);
+
+  onMounted(() => {
+    // should always be HTMLElement in onMounted if passed a valid div
+    // TODO: handle case when passed invalid div
```

Could you remove this TODO comment and open an issue for it?

---

Feel free to leave the TODO there, but please link to the line of code in the issue that is opened.

---

@mcovalt based on your most recent commit (043ce0d) it would appear as though there's no need to open an issue, since the typing and null check are in place. Am I reading that right?

---

Thanks for making a note of this. I went ahead and opened an issue: https://github.com/atlascope/atlascope/issues/4.

@naglepuff The aforementioned issue clarifies the `TODO` that was here. In summary, if passed a reference to an invalid DOM node, the reference will have a `null` value even in `onMounted`. We want to remove the silent failure this code allows.